roles/lib_openshift/src/class/oadm_policy_group.py

```diff
@@ -0,0 +1,180 @@
+
+        results = self._get(self.config.kind, resource_name)
+        if results['returncode'] == 0:
+            return results
+
+        # Try -binding naming convention as well
+        return self._get(self.config.kind, resource_name + "-binding")
+
+    def exists_role_binding(self):
+        ''' return whether role_binding exists '''
+        results = self.get()
+        if results['returncode'] == 0:
+            self.role_binding = RoleBinding(results['results'][0])
+            if self.role_binding.find_group_name(self.config.config_options['group']['value']) != None:
+                return True
+
+            return False
+
+        elif '\"%s\" not found' % self.config.config_options['name']['value'] in results['stderr']:
+            return False
+
+        return results
+
+    def exists_scc(self):
+        ''' return whether scc exists '''
+        results = self.get()
+        if results['returncode'] == 0:
+            self.security_context_constraint = SecurityContextConstraints(results['results'][0])
+
+            if self.security_context_constraint.find_group(self.config.config_options['group']['value']) != None:
+                return True
+
+            return False
+
+        return results
+
+    def exists(self):
+        '''does the object exist?'''
+        if self.config.config_options['resource_kind']['value'] == 'cluster-role':
+            return self.exists_role_binding()
+
+        elif self.config.config_options['resource_kind']['value'] == 'role':
+            return self.exists_role_binding()
+
+        elif self.config.config_options['resource_kind']['value'] == 'scc':
+            return self.exists_scc()
+
+        return False
+
+    def perform(self):
+        '''perform action on resource'''
+        cmd = ['-n', self.config.namespace, 'policy',
+               self.config.config_options['action']['value'],
+               self.config.config_options['name']['value'],
+               self.config.config_options['group']['value']]
+
+        return self.openshift_cmd(cmd, oadm=True)
+
+    @staticmethod
+    def run_ansible(params, check_mode):
+        '''run the idempotent ansible code'''
+
+        state = params['state']
+
+        action = None
+        if state == 'present':
+            action = 'add-' + params['resource_kind'] + '-to-group'
+        else:
+            action = 'remove-' + params['resource_kind'] + '-from-group'
+
+        nconfig = OadmPolicyGroupConfig(params['namespace'],
+                              params['kubeconfig'],
+                              {'action': {'value': action, 'include': False},
+                               'group': {'value': params['group'], 'include': False},
+                               'resource_kind': {'value': params['resource_kind'], 'include': False},
+                               'name': {'value': params['resource_name'], 'include': False},
+                              })
+
+        oadmpolicygroup = OadmPolicyGroup(nconfig)
```

Comment:
oadm rename
please add the verbose param to the class

`policygroup = PolicyGroup(nconfig, params['debug'])`